spec/dummy/db/schema.rb

```diff
@@ -8,43 +9,42 @@
 # from scratch. The latter is a flawed and unsustainable approach (the more migrations
 # you'll amass, the slower it'll run and the greater likelihood for issues).
 #
-# It's strongly recommended that you check this file into your version control system.
+# It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20140113233821) do
+ActiveRecord::Schema.define(:version => 20140113233821) do
 
```

Comment:
Why the switch to the hash rocket syntax?

Reply:
Because rails didn't introduce the new hash syntax in the schema generation until rails 4.1, but the dev environment is now using rails 3.2. I didn't add any migrations as part of this pull request, but I did have to run `db:test:prepare`, which dumps a new schema file in order to load it into the test db.

Reply:
🐺